.github/bot_templates/ASF_UPGRADE_PR.md

```diff
@@ -0,0 +1,24 @@
+# 🚀 ASF Update Report 🚀
+This PR has been automatically generated to update the generated API stubs for our ASF services.
+It uses the latest code-generator from the _master_ branch ([scaffold.py](https://github.com/localstack/localstack/blob/master/localstack/aws/scaffold.py)) and the latest _published_ version of [botocore](https://github.com/boto/botocore) to re-generate all API stubs which are already present in the `localstack.aws.api` module of the _master_ branch.
+
+## 🔄 Updated Services
+This PR updates the following services:
+{{ SERVICES }}
+
+![Update all the APIs!](http://www.quickmeme.com/img/6f/6fb000d22725df49fad779eae40a28e41745e30a99791889b6ea8db9fa5a32eb.jpg)
```

Comment:
while i'm totally in favor of memes, maybe it'll get annoying after the second or third PR :D